### ISSUE TITLE
Post build event - adding files to path inside .nupkg. 

### ISSUE THREAD

**Issue by katsuragi545** [nuproj/nuproj#270]
I have some files I need to write to a specific path in my .nupkg file.  After a successful build, I would like to place several files into a specific path (runtimes/win7-x64/native) instead of the default 'lib/net45' folder. 

Right now, I have to open the .nupkg file (I use Nuget Package Explorer) and manually create the (runtimes/win7-x64/native) folder structure then add the appropriate files. Note - these files have to be in this specific path in order for them to work in my project. 

Basically, how do I go about using nuproj to handle this for me and eliminate the need for me to do it manually? I've attached an image to better illustrate my question (the part in red is what I'm trying to automate).  

![image](https://cloud.githubusercontent.com/assets/19474503/19395901/ee2822ee-91f4-11e6-8462-abad150eb99b.png)

**Comment by AArnott** [comment]
This is how I add extra files to the nupkg before it's created. In this example, I'm copying assemblies that were placed in the lib\portable- folder into lib\dotnet as well. But you can simply add File items with TargetPath metadata to do whatever you want.

```xml
  <Target Name="CopyAssemblyToLibDotNetFolder" DependsOnTargets="ConvertItems" AfterTargets="ConvertItems">
    <!-- NuProj doesn't yet support populating the lib\dotnet folder. So we do it ourselves. -->
    <ItemGroup>
      <File Include="@(File)"
            Condition=" '%(File.TargetFrameworkMoniker)' == '.NETPortable,Version=v4.5,Profile=Profile111' ">
        <TargetPath Condition=" '%(File.WithCulture)' != 'true' ">lib\dotnet\%(FileName)%(Extension)</TargetPath>
        <TargetPath Condition=" '%(File.WithCulture)' == 'true' ">lib\dotnet\%(File.Culture)\%(FileName)%(Extension)</TargetPath>
        <TargetPath Condition=" '%(File.MSBuildSourceTargetName)' == 'SatelliteDllsProjectOutputGroup' ">lib\dotnet\$([System.IO.Path]::GetFileName($([System.IO.Path]::GetDirectoryName(%(Identity)))))\%(FileName)%(Extension)</TargetPath>
      </File>
    </ItemGroup>
  </Target>
```
